### ISSUE TITLE
v2.7.0 Release Candidate

### ISSUE THREAD

**Issue by Ricks-Lab** [Ricks-Lab/amdgpu-utils#44]
This release candidate include modifications required to facilitate a debian package release.  It also supports running from a cloned local repository.   Please report any bugs or feedback on this release candidate.  Thanks!

**Comment by Ricks-Lab** [comment]
Will potentially deprecate amdgpu-pciid as there is already a version of the decode available with lspci and can be updated with [update-pciids](https://linux.die.net/man/8/update-pciids).  Thanks to @smoe for pointing it out.

**Comment by Ricks-Lab** [comment]
I have remove the repository pciid file and now reference the system's file.  If the user created a local version of the file, with *amdgpu-pciid --install* it will be used instead of the system file and should be removed with *amdgpu-pciid --remove_local*

I had to make significant changes through-out the code, so verification with your, @csecht , us conditions would be appreciated.  Thanks!

**Comment by csecht** [comment]
I just now downloaded the Master, and ran * amdgpu-pciid --remove_local*. But that removed the  correct display card model name. For my RX 570, *amdgpu-ls*  gives:
```
Device ID: {'vendor': '0x1002', 'device': '0x67df', 'subsystem_vendor': '0x1682', 'subsystem_device': '0xc570'}
GPU Frequency/Voltage Control Type: 1
Decoded Device ID: Ellesmere [Radeon RX 470/480/570/570X/580/580X]
Card Model:  Ellesmere [Radeon RX 470/480/570/570X/580/580X] (rev ef)
Short Card Model:  RX 470/480/570/570X/580/580X
Display Card Model:  RX 470/480/570/570X/580/580X
```
Did I miss a step somewhere to install the new/correct pciid file?

**Comment by csecht** [comment]
Nevermind. I just read the updated User Guide (duh), and ran *sudo update-pciids*.  The decoded name now appears in -monitor, -ls, etc.

**Comment by Ricks-Lab** [comment]
Thanks for checking it out. I have released v2.7.0. This will also be used for a Debian package release.
